.github/workflows/linux_build_tests.yaml

```diff
@@ -23,23 +23,19 @@ on:
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
     strategy:
-      max-parallel: 4
       matrix:
         package: [lz4, mpich, tut, py-setuptools, openjpeg, r-rcpp]
     steps:
     - uses: actions/checkout@v2
-    - name: Cache ccache's store
-      uses: actions/cache@v1
+    - uses: actions/cache@v2
```

Comment:
Are these version updates necessary to achieve the effects mentioned in the PR description?

Reply:
No, these are just "maintenance" updates to the actions we use - since I was at it I just bumped versions of all the ones that were lagging behind the latest stable.